src/components/Login/Login.tsx

```diff
@@ -12,69 +11,56 @@ import { CustomResponseResult } from 'src/shared/models/http';
 import { RoutePath } from 'src/shared/enums/router';
 import { user } from 'src/assets/mocks/user';
 import { User } from 'src/shared/models/user';
+import * as Yup from 'yup';
+import { Formik, Form, Field, ErrorMessage } from 'formik';
+
+const validationLoginSchema = Yup.object().shape({
```

Comment:
It is a good idea to have validation schemas on separate files for reusability and cleanliness on prod projects